app/Resources/views/themes/tastefulplanet/blocks/map.html.twig

```diff
@@ -8,69 +8,12 @@
 
         <div class="title">Destinations</div>
 
-        <div class="places-map"></div>
+        <div class="map-container"></div>
         <div class="explore">
             <a href="{{ path(location) }}" class="btn btn-primary btn-lg btn-block">{{ 'Explore destinations'|trans }}</a>
         </div>
     </div>
 
-    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
-          integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="
-          crossorigin=""/>
+    {% include ':themes/tastefulplanet/parts:map.html.twig' with {'contentArray': contentArray} only %}
```

Comment:
please change it to `{% include '@ezdesign/parts/map.html.twig' with {'contentArray': contentArray} only %}`